package.json

```diff
@@ -78,7 +80,7 @@
     "oae-mixpanel": "",
     "oae-preview-processor": "",
     "oae-principals": "",
-    "oae-rest": "latest",
+    "oae-rest": "git://github.com/Coenego/oae-rest#mixpanel-rest-api",
```

Comment:
The direct path needs to be taken out and any necessary PRs merged to the main oae-rest repo.